lib/base/error-handler.js

```diff
@@ -81,13 +82,17 @@ const log = require('npmlog');
             "EXISTING_FILETYPE_PATH" : "A file with the same name at the specified location already exists",
             "NOT_DIRTYPE_PATH" : "The specified path is not a directory",
             "NOT_EXIST_PATH" : "The specified path does not exist",
+            "NOT_EXIST_DAEMONNAME" : "The specified daemon does not exist",
             "NOT_EXIST_SSHKEY_PASSWD" : "Private key file or password does not exist",
             "NOT_EXIST_DISPLAY" : "No existing displayId from getSessionList",
             "NOT_SUPPORT_SESSION" : "This device does not support multiple sessions",
             "NOT_SUPPORT_ENYO" : "Enyo app packaging is not supported",
             "NOT_SUPPORT_AUTHTYPE" : "Not supported auth type",
             "NOT_SUPPORT_RUNNINGLIST" : "Not supported method to get running app information",
+            "NOT_SUPPORT_JOURNALD" : "Journal daemon do not suppport the option",
+            "NOT_SUPPORT_PMLOGD" : "Pmlog daemon do not suppport the option",
```

Comment:
Please fix typo : 
Pmlog daemon does not support the option

Reply:
ok